extension/index.googleFunction.js

```diff
@@ -6,29 +6,39 @@ const auth = require('./src/validator/authentication')
 const logger = utils.getLogger()
 
 exports.extensionTrigger = async (request, response) => {
+  const obj = request?.body?.resource?.obj
```

Comment:
Would be nice if those consistent between handlers?

```suggestion
  const paymentObj = request?.body?.resource?.obj
```

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/630/commits/83cbc9ac399631e7308e35593a0b33d3d3036d48